src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 
-const App: React.FC = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App: React.FC = () => {
+  const [selectedUser, setSelectedUser] = useState('');
+  const [selectedPostId, setSelectedPostId] = useState<number | null>(null);
```

Comment:
use 0 instead of null and remove typing for primitive
```suggestion
  const [selectedPostId, setSelectedPostId] = useState(0);
```